src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -11066,890 +10803,769 @@ void WriteVeriSumTable(EnergyPlusData &state)
-                                }
-                            } else if ((curAzimuth >= 45.0) && (curAzimuth < 135.0)) {
-                                wallAreaE += curArea * mult;
-                                if (isConditioned) wallAreaEcond += curArea * mult;
-                                if (isAboveGround) {
-                                    aboveGroundWallAreaE += curArea * mult;
-                                    if (isConditioned) aboveGroundWallAreaEcond += curArea * mult;
-                                }
-                            } else if ((curAzimuth >= 135.0) && (curAzimuth < 225.0)) {
-                                wallAreaS += curArea * mult;
-                                if (isConditioned) wallAreaScond += curArea * mult;
-                                if (isAboveGround) {
-                                    aboveGroundWallAreaS += curArea * mult;
-                                    if (isConditioned) aboveGroundWallAreaScond += curArea * mult;
-                                }
-                            } else if ((curAzimuth >= 225.0) && (curAzimuth < 315.0)) {
-                                wallAreaW += curArea * mult;
-                                if (isConditioned) wallAreaWcond += curArea * mult;
-                                if (isAboveGround) {
-                                    aboveGroundWallAreaW += curArea * mult;
-                                    if (isConditioned) aboveGroundWallAreaWcond += curArea * mult;
-                                }
+        for (int iSurf = 1; iSurf <= state.dataSurface->TotSurfaces; ++iSurf) {
+            // only exterior surfaces including underground
+            auto const &thisSurf = state.dataSurface->Surface(iSurf);
+            if (!thisSurf.HeatTransSurf) continue;
+            bool const isAboveGround = (thisSurf.ExtBoundCond == ExternalEnvironment) || (thisSurf.ExtBoundCond == OtherSideCondModeledExt);
+            if (isAboveGround || (thisSurf.ExtBoundCond == Ground) || (thisSurf.ExtBoundCond == GroundFCfactorMethod) ||
+                (thisSurf.ExtBoundCond == KivaFoundation)) {
+                Real64 curAzimuth = thisSurf.Azimuth;
+                // Round to two decimals, like the display in tables
+                curAzimuth = round(curAzimuth * 100.0) / 100.0;
+                Real64 curArea = thisSurf.GrossArea;
+                if (thisSurf.FrameDivider != 0) {
+                    Real64 const frameWidth = state.dataSurface->FrameDivider(thisSurf.FrameDivider).FrameWidth;
+                    Real64 const frameArea =
+                        (thisSurf.Height + 2.0 * frameWidth) * (thisSurf.Width + 2.0 * frameWidth) - (thisSurf.Height * thisSurf.Width);
+                    curArea += frameArea;
+                }
+                int const zonePt = thisSurf.Zone;
+                auto const &thisZone = state.dataHeatBal->Zone(zonePt);
```

Comment:
@jmarrec There's an integration test failure here (and now in develop) with DaylightingDeviceShelf, `Zone = 0` for surface Outside Shelf. Since `thisZone` is only used to calculate `mult` in various surface type blocks, I think that can be moved up here and protected with `zonePt > 0`.